### DISCUSSION TITLE
Snap package for ghostty

### DISCUSSION THREAD

**Discussion by ronit18** [ghostty-org/ghostty#3896] [Feature Requests, Ideas] [answered]
if no one else is working on this. Can i create snap package for ghostty, this way it would be very easy to install ghostty on debian distro(i.e ubuntu)

**Comment by JakeWharton** [comment, marked as answer]
See https://github.com/ghostty-org/ghostty/discussions/3153

Probably easiest to coordinate with others working on packaging for various Linux mechanisms on the Discord.

**Comment by Flimm** [comment]
Here's a pull request adding Snap packaging: https://github.com/ghostty-org/ghostty/pull/3931 Also, see this discussion: https://github.com/ghostty-org/ghostty/discussions/3153

**Comment by mikemadden42** [comment]
It does look like there's a snap of ghostty already.
https://snapcraft.io/ghostty

Would it make sense to mention it as an option here?
https://ghostty.org/docs/install/binary

**Reply by rhodes-b** [reply]
This is an not official snap from Mitchell
https://github.com/ghostty-org/ghostty/pull/3931#issuecomment-2569454101
They were going to file a dispute for that name and someone from canonical was interested in maintaining the snap in tree

**Reply by jcollie** [reply]
Ken VanDine is the person from Canonical... The name dispute has already occurred. Once some bugs are ironed out with the provider we run all of our CI jobs on I believe that the snap packages will be produced and owned by Ghostty.
